### PR TITLE
docs: add installation tutorials for skypilot serving

### PR DESCRIPTION
**Pull request by wsxiaoys**
cc @concretevitamin for review

### CODE REVIEW

File: website/docs/installation/skypilot/index.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# Skypilot Serving
+[SkyPilot](https://skypilot.readthedocs.io/en/latest/) is a framework for running LLMs, AI, and batch jobs on any cloud, offering maximum cost savings, highest GPU availability, and managed execution.
+
+[SkyServe](https://skypilot.readthedocs.io/en/latest/serving/sky-serve.html) is SkyPilot’s model serving library. SkyServe (short for SkyPilot Serving) takes an existing serving framework and deploys it across one or more regions or clouds.
```
Comment (concretevitamin) [inline website/docs/installation/skypilot/index.md:4]: I'm wondering if it makes sense to mention the self-host / BYOC benefit somewhere. A sketch:

With this deployment mode, all replica VM(s) running Tabby and a lightweight controller VM are entirely deployed in your own cloud accounts and VPCs.

File: website/docs/installation/skypilot/index.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+## Launch the service
+
+We first execute `sky serve up tabby.yaml -n tabby`.
+
+![start tabby service](./start-service.png)
+
+If everything goes well, you'll see messages below
+![service ready](./service-ready.png)
+
+This only initiates SkyServe's scheduler and load balancer; the actual Tabby service is not yet activated.
```
Comment (concretevitamin) [inline website/docs/installation/skypilot/index.md:44]:
```suggestion
This finishes launching SkyServe's control VM which runs a load balancer for this serve; the actual replica running the Tabby service is undergoing provisioning.
```

**Review by concretevitamin — APPROVED** [review]
Amazing @wsxiaoys! Works like a charm. LGTM

File: website/docs/installation/skypilot/index.md (hunk 1 above)
Comment (wsxiaoys) [reply website/docs/installation/skypilot/index.md:4]: Added

**Pull request merged**